Application/EdFi.Ods.Features/Controllers/TokenInfoController.cs

```diff
@@ -78,7 +80,7 @@ private async Task<IActionResult> GetTokenInformation(TokenInfoRequest tokenInfo
 
             var tokenInfo = await _tokenInfoProvider.GetTokenInfoAsync(apiContext);
 
-            Response.Headers.Add("Cache-Control", "no-cache");
+            Response.GetTypedHeaders().CacheControl = new CacheControlHeaderValue { NoCache = true };
```

Comment:
We also add `[ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]` to the class

Reply:
added